### PR TITLE
feat: use mdcrenderer component and parsemarkdown util outside nuxt

### PR DESCRIPTION
**Pull request by adamdehaven**
Allow utilizing the `MDCRenderer.vue` component and `parseMarkdown` utility in a normal Vue app, outside Nuxt. 

I've updated the `@nuxtjs/mdc` module in this PR along with updates to the primary README for usage instructions.

I've also created a [CodeSandbox demo](https://codesandbox.io/p/devbox/mdc-in-vue-vd7thh) by building the package, running `npm pack`, and importing the changes into the sandbox via a local `nuxtjs-mdc-local.tgz` bundle.

MDCRenderer component rendering markdown and MDC

Please see the README file in the PR as well as the CodeSandbox for a usage example.

@farnabaz @antfu I'd appreciate some eyes on this; mainly around:

1. Is defining import aliases in the host project's Vite config the best way to get around the Nuxt-specific imports that aren't needed? I don't like the host app having to include the aliases and the stubbed import file, but wasn't sure of another way of getting around Vite choking on the import paths. This is a bit fragile, as if any new imports are added in the future, a host project would have to add more aliases to its Vite config. I thought about refactoring at least these internal `#mdc-*` imports but wasn't sure of the downstream impact. 
2. I have syntax highlighting working by exporting a few of the remark and shiki utils that were previously internal (as @farnabaz and I discussed last week at Vue Amsterdam). Syntax highlighting works via the host project defining it's highlighter via the `createShikiHighlighter` util along with the `rehypeHighlight` and passing it into the `parseMarkdown` utility. This works great in my opinion, but just want to make sure this is good as a long-term solution?
3. I have a TODO question in the fallback interface I'm providing in the `MDCRenderer.vue` setup function.

When merged, this will resolve https://github.com/nuxt-modules/mdc/issues/155.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -262,6 +267,120 @@ export default defineNuxtConfig({
...
+::
+`
+
+const ast = ref(null)
+
+onBeforeMount(async () => {
+  ast.value = await parseMarkdown(md, {
+    rehype: {
+      plugins: {
+        shikiji: {
```
Comment (farnabaz) [inline README.md:351]:
```suggestion
        highlight: {
```

**Comment by farnabaz** [comment]
> I can put together an example in the docs, but I couldn't get the docs site to work for me locally. What am I missing?

Hmm, let's ignore it README is fine for now :)

**Comment by farnabaz** [comment]
@adamdehaven I've updated your PR, do you mind checking in your project to confirm functionality?

**Comment by adamdehaven** [comment]
I can confirm the changes work in my project locally, and I have also updated the [Code Sandbox](https://codesandbox.io/p/devbox/mdc-in-vue-vd7thh) with the latest changes. 

I also pushed a couple updates to the README (couple typos, and a few words on the new implementation).

**Comment by adamdehaven** [comment]
@farnabaz any idea when the npm release will be published?

**Comment by farnabaz** [comment]
I made a little change in runtime exports and just released new version

**Pull request merged**